tests/resources/VCH-Util.robot

```diff
@@ -311,6 +311,19 @@ Install VIC Appliance To Test Server With Current Environment Variables
     Log To Console  \nInstalling VCH to test server...
     ${output}=  Run VIC Machine Command  ${vic-machine}  ${appliance-iso}  ${bootstrap-iso}  ${certs}  ${vol}  ${debug}  ${additional-args}
     Log  ${output}
+
+    # Check for failure to obtain IP address
+    # Example:
+    # State of all interfaces:
+    #   "public" IP: "waiting for IP"
+    #   "client" IP: "waiting for IP"
+    #   "management" IP: "waiting for IP"
+    #   "bridge" IP: "waiting for IP"
+    ${noIP}=  Run Keyword If  '"waiting for IP"' in ${output}  Set variable  TRUE
+    ...       ELSE                                             Set Variable  FALSE
```

Comment:
${noIP}=  Set Variable If  '"waiting for IP"' in ${output}  TRUE  FALSE

Reply:
fixed